extensions/resteasy-classic/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/JaxRsSecurityConfig.java

```diff
@@ -1,22 +1,24 @@
 package io.quarkus.resteasy.runtime;
 
-import java.util.List;
-import java.util.Optional;
-
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithName;
+
+import java.util.List;
+import java.util.Optional;
 
 /**
  * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
  */
-@ConfigRoot(name = "security.jaxrs", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
-public class JaxRsSecurityConfig {
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+@ConfigMapping(prefix = "security.jaxrs")
```

Comment:
That is not same, it is not one to one. you need `quarkus.security.jaxrs` unless there is some new feature I missed

Reply:
Yes, correct.

We know it is annoying. Let me provide an explanation for it:

`@ConfigRoot` would automatically add `quarkus` to `ConfigRoot#name` and this was fine for a while. We we started to have external extensions and even Quarkiverse, it would mean that these extension would be force to use the `quarkus` prefix, even if they were not part of the core, so we ended up adding `ConfigRoot#prefix`, for extensions to override the `quarkus` prefix.

When we did the `@ConfigMapping`, we just created a simple parameter `prefix` so everyone could just set the namespace they want.

Reply:
I never realized that was the reason, just took it as "that's how it is". It makes sense, thanks.